### DISCUSSION TITLE
Colormode transition

### DISCUSSION THREAD

**Discussion by insuusvenerati** [system-ui/theme-ui#1159] [General] [answered]
How would one go about transitioning between colormodes slowly?

**Comment by lachlanjc** [comment, marked as answer]
Inspired by one of Brent’s projects, you can easily implement the page background changing like this: https://github.com/lachlanjc/notebook/blob/glitch/src/gatsby-plugin-theme-ui/index.js#L83-L85

**Reply by marelix2** [reply]
could u provide this solution one more time? Got 404 once im pressing link

**Reply by marelix2** [reply]
nvm, i've just added this piece of code to theme  C:
` styles: {
    root: {
      transition: 'background-color 0.5s ease',
    },`

**Reply by lachlanjc** [reply]
I deleted that code (& file) since I decided the effect was actually not feeling very polished, inspired by [this article](https://paco.me/writing/disable-theme-transitions).

[Here’s my new theme switcher code](https://github.com/lachlanjc/notebook/blob/glitch/components/layout.js#L11) if you’re curious!
